feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/moods/AddNewMoodActivity.java

```diff
@@ -233,13 +233,11 @@ public Bitmap byteArrayToBitmap(byte[] input) {
     public void Save(View view) {
         if (saveLocation) {
 
-            if (!checkLocationPermission()) {
-            } else {
+            if (checkLocationPermission()) {
                 moodHistory = currentUser.getMoodHistory();
```

Comment:
wait aren't we supposed to scrap 
moodHistory = currentUser.getMoodHistory()
myMood = getMoodDetails();
etc...
this from the else?

Reply:
AndroidStudio suggests it's still being used, so we should likely keep this